docs/GettingStarted.rst

```diff
@@ -1395,138 +1395,11 @@ You will be then directed to your **Activity Stream** page.
 
 .. _Share-AS:
 
+.. _News-composer:
+
 Sharing in the activity stream
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Using the acivity stream, you are able to share with your connections or
```

Comment:
Now that we have removed the old composer, the developer guide must also be updated, especially this chpater : https://docs.exoplatform.org/en/latest/Application.html?#plfdevguide-developingapplications-extendingexoapplications-applicationplugins-activitycomposer

Reply:
will be treated separately  in this task: https://community.exoplatform.com/portal//spaces/exo_product_management/tasks/taskDetail/25007